### PR TITLE
Script converter metrics comparison

### PR DESCRIPTION
**Pull request by t0yv0**
I'd like to add this script temporarily to keep it checked in while we're rolling out the new PULUMI_EXPERIMENTAL converter. It will help computing the metrics difference between baseline/experimental, something I can attach to PRs during rollout.

### CODE REVIEW

**Review by iwahbe — APPROVED** [review]
Looks good. Can we add a comment explaining that the script is temporary or put it in `unstable/scripts/...`?

Tests look like they are failing because of https://github.com/pulumi/ci-mgmt/pull/537 (fixed by https://github.com/pulumi/pulumi-terraform-bridge/pull/1350).

**Comment by t0yv0** [comment]
https://github.com/pulumi/pulumi-terraform-bridge/issues/1280 is the context. Yes will do.

File: unstable/scripts/exconverter/main.go [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+		s.exampleByHCL[entry.ExampleName] = entry
+	}
+
+	ss, err := os.ReadFile(filepath.Join(dir, "shortSummary.txt"))
+	noerr(err)
+	s.shortSummary = string(ss)
+	return s
+}
+
+func tfgen(convert int) {
```
Comment (iwahbe) [inline unstable/scripts/exconverter/main.go:153]: I'm getting strong C vibes here. Why isn't convert a `bool`?

File: unstable/scripts/exconverter/main.go [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+	s := stats{
+		exampleByHCL: map[string]flattenedExample{},
+	}
+
+	dec := json.NewDecoder(f)
+	for {
+		var entry flattenedExample
+		err := dec.Decode(&entry)
+		if err != nil || entry.ExampleName == "" {
+			break
```
Comment (iwahbe) [inline unstable/scripts/exconverter/main.go:141]: It feels like this should be a `continue`. Why are we stopping on the first error?
Why don't we report the error?

**Review by iwahbe — APPROVED** [review]
It looks like this script needs to be run in the directory of the provider that it is testing. That would be an excellent thing to put in a comment.

**Pull request merged**